### PR TITLE
add get_env function to yuck

### PR DESCRIPTION
**Pull request by RegenJacob**
## Description

This PR adds the function `get_env("KEY")` to yuck.
SCSS already has the ability to read env vars so adding it directly to yuck seemed nice

When the specified env var is empty it returns an empty string 

## Usage

```fennel
(label :text "${get_env("USER")}")
```

## Additional Notes

This needs more testing.

## Checklist

Please make sure you can check all the boxes that apply to this PR.

- [x] I added my changes to CHANGELOG.md, if appropriate.
- [x] The documentation in the `docs/content/main` directory has been adjusted to reflect my changes.
- [x] I used `cargo fmt` to automatically format all code before committing

### CODE REVIEW

File: docs/src/expression_language.md [hunk 1]
```diff
@@ -47,3 +47,4 @@ Supported currently are the following features:
 	- `arraylength(value)`: Gets the length of the array
 	- `objectlength(value)`: Gets the amount of entries in the object
 	- `jq(value, jq_filter_string)`: run a [jq](https://stedolan.github.io/jq/manual/) style command on a json value. (Uses [jaq](https://crates.io/crates/jaq) internally).
+    - `get_env(string)`: Gets the specified Enviroment variable
```
Comment (elkowar) [inline docs/src/expression_language.md:50]:
```suggestion
    - `get_env(string)`: Gets the specified environment variable
```

File: crates/simplexpr/src/eval.rs [hunk 2]
```diff
@@ -307,6 +307,14 @@ impl SimplExpr {
...
 fn call_expr_function(name: &str, args: Vec<DynVal>) -> Result<DynVal, EvalError> {
     match name {
+        "get_env" => match args.as_slice() {
+            [string] => {
+                let string = string.as_string()?;
+                let var = std::env::var(string).unwrap_or("".to_string());
+                Ok(DynVal::from(var))
+            }
+            _ => Err(EvalError::WrongArgCount(name.to_string())),
+        },
```
Comment (elkowar) [inline crates/simplexpr/src/eval.rs:317]:
```suggestion
        "get_env" => match args.as_slice() {
            [var_name] => {
                let var = std::env::var(var_name.as_string()?).unwrap_or_default();
                Ok(DynVal::from(var))
            }
            _ => Err(EvalError::WrongArgCount(name.to_string())),
        },
```

**Review by elkowar — CHANGES_REQUESTED** [review]
Some slight cleanup, but otherwise this looks good!

**Comment by RegenJacob** [comment]
I changed the code accordingly

**Comment by RegenJacob** [comment]
Hi @elkowar ,

Just a friendly ping regarding this pull request submitted several months ago. It seems to be sitting idle despite being issue-free. If there are any concerns or additional steps needed, please let me know. Appreciate your attention to this.

**Comment by elkowar** [comment]
Thanks for the ping, completely forgot to check this pr again! Thanks for your contribution!

**Pull request merged**